docs/src/cookbook_create_user_programmatically.py

```diff
@@ -21,5 +21,7 @@ async def create_user(email: str, password: str, is_superuser: bool = False):
                         )
                     )
                     print(f"User created {user}")
+                    return user
     except UserAlreadyExists:
         print(f"User {email} already exists")
+        raise UserAlreadyExists
```

Comment:
Could you change it to just `raise`? It means that we just "re-raise" the exception instead of building another.

Reply:
Done.